Musicca/app/src/main/java/com/example/musicca/fragments/PeopleFragment.java

```diff
@@ -70,20 +70,24 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
             public void onRefresh() {
                 // Remember to CLEAR OUT old items before appending in the new ones
                 peopleAdapter.clear();
-                queryUsers(0);
+                queryUsers();
                 peopleAdapter.notifyDataSetChanged();
             }
         });
     }
 
-    protected void queryUsers(int page) {
+    protected void queryUsers() {
         ParseQuery<ParseUser> query = ParseUser.getQuery();
         query.findInBackground(new FindCallback<ParseUser>() {
-            public void done(List<ParseUser> objects, ParseException e) {
+            public void done(List<ParseUser> parseUsers, ParseException e) {
                 if (e == null) {
                     // The query was successful.
-                    allUsers.addAll(objects);
-
+                    // removing current user from the list
+                    for (int i = 0; i < parseUsers.size(); i++) {
+                        if (parseUsers.get(i).getUsername() != ParseUser.getCurrentUser().getUsername()){
+                            allUsers.add(parseUsers.get(i));
+                        }
```

Comment:
I also ensure that current user is not added to the list of users being bound by the adapter to the fragment's recycler view